mathsnuggets/widgets/taylor.py

```diff
@@ -0,0 +1,19 @@
+import sympy
+
+from mathsnuggets.core import fields, form
+
+test = {"function": "x sin x"}
```

Comment:
Remove this, the test is not adapted for this widget

Reply:
Okay